CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+Development process
```

Comment:
Please take a look at the previous contributing guides for inspiration:
https://github.com/redhat-beyond/AcadeMe/blob/main/contributing.md
https://github.com/redhat-beyond/Xpense/blob/main/CONTRIBUTING.md